erc20/0xaefb534eAEF097C41c28b58699aDb7903C602B98.json

```diff
@@ -0,0 +1,19 @@
+{
+  "symbol": "CIB",
+  "address": "0xaefb534eAEF097C41c28b58699aDb7903C602B98",
+  "overview":{
+        "en": "The cultural industry basic chain (CIB) is a comprehensive blockchain system of “multi-chain coexistence and cross-chain integration”. It takes the cultural industry as the entrance, and uses the brand-new concepts and technologies of the blockchain to The needs, pain points, difficulties, and development bottlenecks of industrial development, build an efficient, easy-to-use, distributed blockchain architecture platform to serve the real economy and meet the real business needs of the real economy. The cultural industry basic chain uses the technology of "multi-chain coexistence and cross-chain fusion" to ensure the feasibility and reliability of the application scenarios. From the start, development, cost, revenue, and repayment of the project are automatically handled by smart contracts, the investment direction is open and transparent, and combined with the traditional culture of the landing enterprise (project), creating a platform that runs through the entire process of investment and management of the cultural film and television industry!",
+        "zh": "CIB是一个“多链并存，跨链融合”的综合区块链体系，它以文化产业为切入口，用区块链全新的理念和技术，对现有文化产业的各方需求及产业发展的痛点、难点、发展瓶颈，搭建起高效、易用、分布式的区块链架构平台，服务于实体经济，满足实体经济的真实商业需求。文化产业基础链用“多链并存，跨链融合”的技术，保障落地应用场景的可行性与可靠性。从项目的开始、发展、成本、收益、回款都由智能合约自动处理，投资去向公开、透明，结合落地企业（项目）传统文化，打造 一个贯穿文化影视产业的投资、管理全流程的平台！"
+  },
+  "email": "cib2020@163.com",
+  "website": "http://r59iak.cn",
+  "whitepaper": "http://www.r59iak.cn/a/chanpin/fushizhubao/62.html",
+  "state": "NORMAL",
+  "published_on": "2019-01-06",
+  "initial_price":{
+        "ETH":"0.00222222 ETH",
+  },
+  "links": {
+    "medium": "https://www.jinse.com/blockchain/533295.html"
+  }
+}
```

Comment:
1636 is the PR number, as in: https://github.com/consenlabs/token-profile/pull/1636 This number is the key to verify yourself as the rightful editor of the information to imToken.
The token information can only be changed after the official verification

Reply:
i have updated the PR number in my website http://r59iak.cn,thank you.